.github/workflows/ci.yml

```diff
@@ -87,12 +65,44 @@ jobs:
             --cache-from "$CORE_CI_IMAGE:branch--$BRANCH_REF" \
             .
       - name: Push dependabot-core-ci image to GHCR
-        env:
-          ACCESS_CANARY: ${{ secrets.ACCESS_CANARY }}
-        if: env.ACCESS_CANARY != ''
         run: |
           docker push "$CORE_CI_IMAGE:latest"
           docker push "$CORE_CI_IMAGE:branch--$BRANCH_REF"
+
+  test:
```

Comment:
Why break out test into another job?

Reply:
At a high-level, the idea is to build a single CI image, and then run tests for each ecosystem on that image. The idea being that building once is better than building separately for each ecosystem.